### PR TITLE
Generated from api spec 2.0.13

### PR DESCRIPTION
**Pull request by alekshs**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tiledb-inc/tiledb-cloud",
-  "version": "0.0.5",
+  "version": "0.0.6",
```
Comment (Shelnutt2) [inline package.json:3]: We released `0.0.6` as the browser version, should this be `0.0.7`?

File: src/README.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ navigate to the folder of your consuming project and run one of the following co
 _published:_
 
 ```
-npm install @tiledb-inc/tiledb-cloud@2.0.2 --save
+npm install @tiledb-inc/tiledb-cloud@2.0.13 --save
```
Comment (SarantopoulosKon) [inline src/README.md:39]: The npm's package version will be 0.0.7 right?

Comment (alekshs) [reply src/README.md:39]: This is auto generated inside src.

Comment (alekshs) [reply src/README.md:39]: Do we really need this auto generated readme? Should we remove it?

**Review by Shelnutt2 — APPROVED** [review]

File: src/README.md (hunk 2 above)
Comment (SarantopoulosKon) [reply src/README.md:39]: tbh i don's see any value having this README i guess we can just remove it, right now it only complicates users.

**Review by SarantopoulosKon — APPROVED** [review]

**Review by Shelnutt2 — APPROVED** [review]

**Pull request merged**